source/Generic/ExtraMetadataToolsVideo/ExtraMetadataToolsVideo.psm1

```diff
@@ -59,15 +59,241 @@ function GetGameMenuItems
     $menuItem11.FunctionName = "Update-AssetsStatusGameDatabase"
     $menuItem11.MenuSection = "Extra Metadata tools"
 
-    return $menuItem, $menuItem2, $menuItem3, $menuItem9, $menuItem10, $menuItem4, $menuItem5, $menuItem6, $menuItem7, $menuItem8, $menuItem11
+    $menuItem12 = New-Object Playnite.SDK.Plugins.ScriptGameMenuItem
+    $menuItem12.Description = [Playnite.SDK.ResourceProvider]::GetString("LOCExtra_Metadata_tools_MenuItemInvoke-ViewSettingsDescription")
+    $menuItem12.FunctionName = "Invoke-ViewSettings"
+    $menuItem12.MenuSection = "Extra Metadata tools"
```

Comment:
I think the menu item should be in the video section since it's specific to that section only. Other option is to add the word "Video" to the settings description, whichever you prefer.

Reply:
Moved to 'Video' menu.
Done in 8718cdfc5830cdd1bb187cd65b3c2bad02bbde44